simulators/ethereum/engine/clmock.go

```diff
@@ -341,6 +341,7 @@ type BlockProcessCallbacks struct {
 	OnForkchoiceBroadcast     func()
 	OnSafeBlockChange         func()
 	OnFinalizedBlockChange    func()
+	OnVerificationEnd         func()
```

Comment:
This addition seems fine to me, but perhaps the name is not accurate. This is more like OnBlockProductionEnd.